background.js

```diff
@@ -1,5 +1,5 @@
 // create an alarm
-chrome.alarms.create('read later notifier', { delayInMinutes : 1, periodInMinutes : 1 });
+chrome.alarms.create('read later notifier', { delayInMinutes : 1, periodInMinutes : 2 });
```

Comment:
Fix 2 to 10

Reply:
Fix in [this commit](https://github.com/kenta7777/read-later-notifier/pull/7/commits/3bd1e22a07f104ef777ad9a4a586aacb6861bc45)